src/Http/RedirectResponse.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Framework3\Http;
+
+class RedirectResponse extends Response
+{
+    /**
+     * @var string
+     */
+    private string $routePath;
+
+    public function __construct(string $routePath, array $data)
+    {
+        parent::__construct();
+        $this->getData()->setAll($data);
+        $this->routePath = $routePath;
+    }
+
+    public function send(): void
+    {
+        $this->setSessionData();
+        header("Location: " . $this->routePath);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6299285)